build.gradle

```diff
@@ -6,40 +8,45 @@ plugins {
     id 'org.jetbrains.kotlin.plugin.jpa'
     id 'java'
     id 'maven-publish'
-    id 'net.corda.plugins.csde'
+    id 'net.corda.gradle.plugin'
 }
 
 allprojects {
-    group 'com.r3.developers.csdetemplate'
+    group 'com.r3.developers.cordapptemplate'
     version '1.0-SNAPSHOT'
 
     def javaVersion = VERSION_17
 
-    // Configure the CSDE
-    csde {
+    // Configure Corda runtime gradle plugin
+    cordaRuntimeGradlePlugin {
+        notaryVersion = cordaNotaryPluginsVersion
+        notaryCpiName = "NotaryServer"
+        corDappCpiName = "MyCorDapp"
+        cpiUploadTimeout = "30000"
+        vnodeRegistrationTimeout = "60000"
+        cordaProcessorTimeout = "300000"
+        workflowsModuleName = "workflows"
         cordaClusterURL = "https://localhost:8888"
+        cordaRestUser = "admin"
+        cordaRestPasswd ="admin"
+        composeFilePath = "config/combined-worker-kafka-compose.yaml"
         networkConfigFile = "config/static-network-config.json"
         r3RootCertFile = "config/r3-ca-key.pem"
-        corDappCpiName = "MyCorDapp"
-        notaryCpiName = "NotaryServer"
-        cordaRpcUser = "admin"
-        cordaRpcPasswd ="admin"
-        workflowsModuleName = workflowsModule
-        csdeWorkspaceDir = "workspace"
-        notaryVersion = cordaNotaryPluginsVersion
-        combinedWorkerVersion = combinedWorkerJarVersion
-        postgresJdbcVersion = "42.4.3"
-        cordaDbContainerName = "CSDEpostgresql"
-        cordaBinDir = "${System.getProperty("user.home")}/$cordaBinariesDirectory"
-        cordaCliBinDir = "${System.getProperty("user.home")}/$cordaCliBinariesDirectory"
-        cpiUploadTimeout = cpiUploadDefault
-        cordaProcessorTimeout = processorTimeout
-        vnodeRegistrationTimeout = vnodeRegistrationTimeoutDefault
-        skipTestsDuringBuildCpis = this.skipTestsDuringBuildCpis
+        skipTestsDuringBuildCpis = "false"
+        cordaRuntimePluginWorkspaceDir = "workspace"
+        cordaBinDir = "${System.getProperty("user.home")}/.corda/corda5"
+        cordaCliBinDir = "${System.getProperty("user.home")}/.corda/cli"
+
+        // Alternatively, you can use a Database-only combined worker:
+        // composeFilePath = "config/combined-worker-compose.yaml"

```

Comment:
We have an example of a Database-only compose file in the plugin's readme; the link to the readme was added to this repo's README. 

The question is - do we still want to provide a no-kafka example, including this comment and the compose yaml file in the config?

Reply:
We should steer people to the Kafka enabled version, as it's quicker. We then offer the db only version to those who need it.

Reply:
I think it's fine to just have the Kafka version here. The non-Kafka version remains documented with the Gradle plugin.